code/modules/mob/living/carbon/xenomorph/logout.dm

```diff
@@ -0,0 +1,3 @@
+/mob/living/carbon/xenomorph/Logout()
+	. = ..()
+	addtimer(CALLBACK(src, .proc/handle_afk_takeover), XENO_AFK_TIMER)
```

Comment:
Add a reference same as the human one, make the timer stoppable and delete it on login?

Reply:
For xenos away_time is used as a number so when xenos spawn with it, they can be taken away instantly.

Reply:
Using the same var to represent two things can get messy.

Reply:
This doesn't need to be a timer_id stoppable since if they get qdeleted it already gets properly unsed, and take a look at the proc it calls, it checks stuff pretty well.

Reply:
That is fine for now. In the future things might change, so having a reference to the timer would be positive.
Even if that's not desired, if we change what the variable represents in one place it should change everywhere. A same variable representing two different things is very confusing.